src/utils/validate.ts

```diff
@@ -1,3 +1,9 @@
 export const isValidUsername = (str: string) => ['admin', 'editor'].indexOf(str.trim()) >= 0;
 
 export const isExternal = (path: string) => /^(https?:|mailto:|tel:)/.test(path);
+
```

Comment:
Are they used somewhere? I previously remove them because they are not been used in the minimal template.